### PR TITLE
[ALLUXIO-2894] Improve Help Command to adjust the output width according to terminal width

### PR DESCRIPTION
**Pull request by xtang2**
https://alluxio.atlassian.net/browse/ALLUXIO-2894

### CODE REVIEW

File: shell/src/main/java/alluxio/shell/command/HelpCommand.java [hunk 1]
```diff
@@ -44,7 +44,9 @@ public static void printCommandInfo(ShellCommand command, PrintWriter pw) {
     String description =
         String.format("%s: %s", command.getCommandName(), command.getDescription());
     // TODO(binfan): instead of using a constant, adjust the value according to console width
-    int width = HELP_FORMATTER.getWidth();
+    // int width = HELP_FORMATTER.getWidth();
+    int width = TerminalFactory.get().getWidth();
```
Comment (calvinjia) [inline shell/src/main/java/alluxio/shell/command/HelpCommand.java]: Remove the old code and the TODO.

File: shell/src/main/java/alluxio/shell/command/HelpCommand.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
 import java.util.Map;
 import java.util.SortedSet;
 import java.util.TreeSet;
-
+import jline.TerminalFactory;
```
Comment (calvinjia) [inline shell/src/main/java/alluxio/shell/command/HelpCommand.java]: This import should go with the other third party imports above.

**Comment by xtang2** [comment]
TerminalFactory can only be used in 2.14.2 version

File: shell/src/main/java/alluxio/shell/command/HelpCommand.java (hunk 1 above)
Comment (xtang2) [inline shell/src/main/java/alluxio/shell/command/HelpCommand.java]: edited

File: shell/src/main/java/alluxio/shell/command/HelpCommand.java [hunk 3]
```diff
@@ -18,12 +18,13 @@
 import org.apache.commons.cli.CommandLine;
 import org.apache.commons.cli.HelpFormatter;
 
+import jline.TerminalFactory;
```
Comment (calvinjia) [inline shell/src/main/java/alluxio/shell/command/HelpCommand.java]: This should be
```
import jline.TerminalFactory;
import org.apache.commons.cli.CommandLine;
import org.apache.commons.cli.HelpFormatter;
```

Comment (xtang2) [inline shell/src/main/java/alluxio/shell/command/HelpCommand.java]: adjusted

**Pull request merged**